bin/i18n/resources/dashboard/course_content/sync_in.rb

```diff
@@ -200,6 +200,26 @@ def process
                   i18n_strings[BLOCK_CATEGORIES_TYPE][name] = name if name
                 end
 
+                ## Lab2 Function Names
+                if level.uses_lab2?
+                  functions = level.
+                    try(:level_data).
+                    try(:[], "startSources").
+                    try(:[], "blocks").try(:[], "blocks")&.
```

Comment:
minor: I think you can do `level.properties.dig("level_data", "startSources", "blocks")`

Reply:
Nice.  Worked with a slight tweak to add the second `"blocks"`.